Libraries/PropGeomCalc/ExtractRINEXeph.m

```diff
@@ -16,10 +16,16 @@
     DDD = ['0',DDD];
 end
 YY = YYYY(3:4);
-RinexFile = [datadir,YYYY,'/',DDD,'/',YY,'n/brdc',DDD,'0.',YY,'n.Z'];
+filename = ['brdc',DDD,'0.',YY,'n.Z'];
+RinexFile = [datadir,YYYY,'/',DDD,'/',YY,'n/',filename];
 [~,ephfile] = fileparts(RinexFile);
 if ~exist(ephfile,'file')
-    if ~exist([ephfile,'.Z'],'file') 
+    if ~exist([ephfile,'.Z'],'file')
+        username = input('Write the username: ', 's');
+        password = input('Write the password: ', 's');
+
+        system(['wget --auth-no-challenge --user=', username ' --password=', password, ' -O ', filename,' ', RinexFile])

```

Comment:
For instance, I've tested on a MacOS I saw that, as Windows, it doesn't have `wget`, so I will need to change this line to be able to download the RINEX file in other way. If I make it out, I will open other PRs :)